compiler/tests/generated/service_attributes_grpc.h

```diff
@@ -38,135 +38,114 @@
 namespace tests
 {
 
-class Foo final
+struct Foo final
 {
-public:
-    struct Schema;
+    struct Schema
+    {
+        static const ::bond::Metadata metadata;
+
+        private: static const ::bond::Metadata s_foo_metadata;
+
+        public: struct service
+        {
+            typedef ::bond::ext::gRPC::reflection::MethodTemplate<
+                Foo,
+                ::bond::bonded< ::tests::Param>,
+                ::bond::bonded< ::tests::Result>,
+                &s_foo_metadata
+            > foo;
+        };
 
-    template <typename TThreadPool>
-    class ClientCore
+        private: typedef boost::mpl::list<> methods0;
+        private: typedef boost::mpl::push_front<methods0, service::foo>::type methods1;
+
+        public: typedef methods1::type methods;
+
+        
+    };
+
+    class Client
     {
     public:
-        ClientCore(
+        Client(
             const std::shared_ptr< ::grpc::ChannelInterface>& channel,
             std::shared_ptr< ::bond::ext::gRPC::io_manager> ioManager,
-            std::shared_ptr<TThreadPool> threadPool);
-
-        void Asyncfoo(::std::shared_ptr< ::grpc::ClientContext> context, const ::bond::bonded< ::tests::Param>& request, const std::function<void(std::shared_ptr< ::bond::ext::gRPC::unary_call_result< ::tests::Result>>)>& cb);
-        void Asyncfoo(::std::shared_ptr< ::grpc::ClientContext> context, const ::tests::Param& request, const std::function<void(std::shared_ptr< ::bond::ext::gRPC::unary_call_result< ::tests::Result>>)>& cb)
+            const ::bond::ext::gRPC::Scheduler& scheduler = {})
```

Comment:
Make sure to resolve this against the scheduler changes so that it drops the `{}`.